### DISCUSSION TITLE
Change chart y-column based on display variable dropdown

### DISCUSSION THREAD

**Discussion by amruthkiran94** [TerriaJS/terriajs#6755] [Q&A] [answered]
Hello!
Would it be possible to update the `featureInfoTemplate` chart's `y-column` (x-column being date for example), based on the display variable dropdown from the side panel? Currently, the columns have to be hard-coded based on the data.
```
 "featureInfoTemplate": {
 "name": "<nameHere>",
  "template": "<chart id='<id>' title='<titleHere>' sources='<sourceHere>' x-column='date' y-column='load' column-names='Date, Load' column-units='Time,Load'></chart>" 
}
```

**Comment by na9da** [comment, marked as answer]
Not possible currently - I have [created an issue](https://github.com/TerriaJS/terriajs/issues/6758) to support this use case.

**Reply by cczablan1** [reply]
Has this been addressed?
